src/js/views/main/chargeMoneyView.ts

```diff
@@ -0,0 +1,54 @@
+import { $ } from '../../utils/common';
+import { chargeMoneyTemplate, sectionTemplate } from '../../templates/main/chargeMoneyTemplate';
+import { CoinsType } from '../../types/types';
+import { SELECTOR } from '../../constants/viewConstants';
+import showSnackbar from '../../utils/snackbar';
+import VendingMachine from '../../vendingMachine/vendingMachine';
+import { CONFIRM_MESSAGE } from '../../constants/confirmConstants';
+
+export default class ChargeMoneyView {
+  constructor(private readonly vendingMachine: VendingMachine) {
+    this.vendingMachine = vendingMachine;
+  }
```

Comment:
```suggestion
  constructor(private readonly vendingMachine: VendingMachine) {}
```
생략하셔도 암묵적으로 초기화를 진행합니다~
[참고](https://poiemaweb.com/typescript-class#3-%EC%83%9D%EC%84%B1%EC%9E%90-%ED%8C%8C%EB%9D%BC%EB%AF%B8%ED%84%B0%EC%97%90-%EC%A0%91%EA%B7%BC-%EC%A0%9C%ED%95%9C%EC%9E%90-%EC%84%A0%EC%96%B8)

Reply:
수정했습니다!